docs/howto/index.md

```diff
@@ -0,0 +1,75 @@
+# How-To Guides
```

Comment:
May need either of: 
- a little text that clarifies that this is all during the build, e.g. _you can't do any of this at runtime_ that have access to the configuration and deployment
- another level of headers, and (a place for) some how-to guides for users of an existing site that only have access to the browser
  - getting files out of contents
  - clearing out local versions
  - etc.

Reply:
Good point. Added a note in https://github.com/jupyterlite/jupyterlite/pull/641/commits/95927241d5a76f7614b68a067d6c67b479e52b51.

> another level of headers, and (a place for) some how-to guides for users of an existing site that only have access to the browser 

We can do this in a follow-up PR, since there has been good progress on https://github.com/jupyterlite/jupyterlite/pull/655 and https://github.com/jupyterlite/xeus-python-kernel.